02Lesson/01Project/index.html

```diff
@@ -1,9 +1,34 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <title></title>
-</head>
-<body>
-
-</body>
+<html style="background-color: powderblue">
+  <head>
+    <title>Motorcycle Taxidermy</title>
+  </head>
+  <body>
+    <center><img src="https://cyclingdynamics.files.wordpress.com/2013/10/mad-cow-motorcycle_wasav_38681.jpg" alt="MadCow Motorcycle. Skeletal Choppers made from real bones!"></center>
+    <p><h1 style="font-family: Verdana; background-color: pink; text-align: center;">Roy's Motorcycle Taxidermy</h1>
+      <h3 style="font-family: Verdana; color: red;">Need a ride?</h3>
+      <h4 style="font-family: Verdana; color: red;">Got a large dead animal??</h4>
+    </p>
+    <p style="font-family: Verdana">
```

Comment:
Use `p` tag only for real text paragraphs. If you want to put more in block, use `div` `section` etc.